.github/workflows/docs.yaml

```diff
@@ -1,9 +1,10 @@
 name: tbdex SDK Publish Docs to GH Pages
 
 on:
-  # Runs on pushes targeting the default branch
+  # Runs on new releases publishing
   push:
-    branches: ["main"]
```

Comment:
Do we have a use case for publishing from a branch other than `main`?

Reply:
Not at the moment. We only have the latest release being hosted at https://tbd54566975.github.io/tbdex-kt/